resources/modernizer_exclusions.txt

```diff
@@ -20,3 +20,4 @@ com/google/common/primitives/Longs.compare:(JJ)I
 com/google/inject/Inject
 com/google/inject/Provider
 java/lang/StringBuffer."<init>":()V
+java/util/Hashtable."<init>":(I)V
```

Comment:
@andrewgaul Note that the signatures of the OSGi framework require a `Dictionary<String, ?>`, and the properties object is no longer compatible, as it extends `Hashtable<Object, Object>`. We need to pass a proper dictionary there, thus this exclusion (`HashMap` does not extend `Dictionary`).